core/os/thread.h

```diff
@@ -135,6 +150,8 @@ class Thread {
 
 	typedef uint64_t ID;
 
+	static constexpr size_t CACHE_LINE_BYTES = sizeof(void *);
```

Comment:
Why does this differ from the definitions above? Is it expected that this would produce a different value?

Reply:
The purpose of this constant is that classes wary of false sharing can be aligned or padded enough to avoid false sharing. Therefore, in a threadless build, where you don't worry about that, it would only cause some memory waste. For threadless, I decided to use a value that would fit the default alignment.

Reply:
Thank you for the clarification. On Github it wasn't clear that this was the NO_THREADS variant

Reply:
@mrsaturnsan, I'm just raising the fallback value to 128. In machines with smaller cache liines, it would just cause a tiny waste of memory.